Update README.md for GA

elraikhm opened the pull request:
Based on the customer feedback need to have a consistent naming and scope for the library releases across different languages

elraikhm committed:
Update README.md for GA

Based on the customer feedback need to have a consistent naming and scope for the library releases across different languages

AlexGhiondea commented on README.md:
```diff
@@ -15,18 +15,21 @@ Each service might have a number of libraries available from each of the followi
+* [Azure.Security.KeyVault.Certificates](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md)
+* [Azure.Messaging.EventHubs](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/README.md)
 * [Azure.Storage.Files](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Files/README.md)
 * [Azure.Storage.Queues](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Queues/README.md)
```
Queues is GA as well.

ShivangiReja replied:
[Storage - Blobs Batch](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs.Batch/README.md) is missing and It's GA.

ShivangiReja commented on README.md:
```diff
@@ -15,18 +15,21 @@ Each service might have a number of libraries available from each of the followi
-The libraries released in the July 2019 preview:
-* [Azure.ApplicationModel.Configuration](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.Data.AppConfiguration/README.md)
-* [Azure.Messaging.EventHubs](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/README.md)
+The libraries released in the November 2019 GA release are:
```
[Core](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/README.md) and [Identity ](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/README.md)are also missing in this release list. Should we mention these here?

elraikhm committed:
Update README.md

Updated per comments, I did not add core to be aligned with Java. Feel free to add if needed
